src/main/java/ch/njol/skript/tests/platform/Environment.java

```diff
@@ -262,9 +264,8 @@ public void run() {
 		}
 
 		int code = process.waitFor();
-		if (code != 0) {
+		if (code != 0)
 			throw new IOException("environment returned with code " + code);
-		}
 
 		// Read test results
 		TestResults results = new Gson().fromJson(new String(Files.readAllBytes(env.resolve("test_results.json"))), TestResults.class);
```

Comment:
Results shouldn't be read in DEV_MODE, right? When testing myself, even when using the `sk test` command, no results are written

Reply:
Oh I think I get what you mean. The file doesn't exist when in dev mode, so it throws an error when closing the server. Fixed.